src/textual/widgets/_button.py

```diff
@@ -70,8 +75,8 @@ def validate_label(self, label: RenderableType) -> RenderableType:
             return Text.from_markup(label)
         return label
 
-    def render(self) -> RenderableType:
-        return self.label
+    def render(self, style: Style) -> RenderableType:
+        return Text.styled(self.label.plain, style)
```

Comment:
This will replace all the styles in the Text object.

Imagine a Text object created with `Text.from_markup("hello [b]World")`. The bold will be removed.

Suggest will apply the style on top of the label. So 

`label = self.label.copy().stylize(style)`

That way if the CSS styles it to italic, it will still have the additional bold style.

Reply:
Ah sorry, think I misunderstood our chat yesterday. Will fix this today.